JS/main.js

```diff
@@ -0,0 +1,157 @@
+const menuActive = document.querySelector('.menuDropDown');
+const menuCross = document.querySelector('.menu-cross');
+const sideBar = document.querySelector('.side-bar');
+const links = document.querySelectorAll('.entity');
+const popUp = document.querySelector('.pop-up--overlay');
+const circle1 = document.querySelector('.circle1');
+const circle2 = document.querySelector('.circle2');
+
+menuActive.addEventListener('click', () => {
+  sideBar.classList.add('active');
+  menuActive.classList.add('active');
+  menuCross.classList.add('active');
+});
+
+menuCross.addEventListener('click', () => {
+  sideBar.classList.remove('active');
+  menuActive.classList.remove('active');
+  menuCross.classList.remove('active');
+});
+
+links.forEach((link) => link.addEventListener('click', () => {
+  sideBar.classList.remove('active');
+  menuActive.classList.remove('active');
+  menuCross.classList.remove('active');
+}));
+
+const cardcontainer = document.querySelector('.lang-sec');
+
+const languages = (string1, string2, string3, string4) => `<li>${string1}</li>
+  <li>${string2}</li>
+  <li>${string3}</li>
+  <li class="ruby">${string4}</li>`;
+
+const cardsinfo = [
+  {
+    reverseCard: 'row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
+  },
+  {
+    reverseCard: 'reverse-row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy tex.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
+  },
+  {
+    reverseCard: 'row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
+  },
+  {
+    reverseCard: 'reverse-row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
```

Comment:
Please add a link to the source code and live version of the project. 👍🏻

Reply:
@zainsadaqat 
Thank you very much for your review and helping me become a better developer.
Happy coding!

Reply:
- [x] This issue is well resolve :heavy_check_mark: